src/openrct2/windows/_legacy.c

```diff
@@ -133,3 +141,427 @@ void game_command_callback_pickup_staff(sint32 eax, sint32 ebx, sint32 ecx, sint
         break;
     }
 }
+
+uint64 _enabledRidePieces;
+uint8 _rideConstructionState2;
+
+// This variable is updated separately from ride->num_stations because the latter
+// is unreliable if currently in station construction mode
+bool _stationConstructed;
+bool _deferClose;
+
+void game_command_callback_place_ride_entrance_or_exit(sint32 eax, sint32 ebx, sint32 ecx, sint32 edx, sint32 esi, sint32 edi, sint32 ebp)
+{
+    audio_play_sound_at_location(
+        SOUND_PLACE_ITEM,
+        gCommandPosition.x,
+        gCommandPosition.y,
+        gCommandPosition.z
+    );
+
+    Ride *ride = get_ride(gRideEntranceExitPlaceRideIndex);
+    if (ride_are_all_possible_entrances_and_exits_built(ride)) {
```

Comment:
This whole file does not adhere to the code style.

Reply:
This is legacy **C** code just moved to a separate file. I'm fine with it staying as-is.

Reply:
And I am not. The code rules apply to everything. You touch code, you fix it.